packages/protocol/migrations/10_1_exchange_EUR.ts

```diff
@@ -0,0 +1,57 @@
+/* tslint:disable:no-console */
+
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import {
+  deploymentForCoreContract,
+  getDeployedProxiedContract,
+} from '@celo/protocol/lib/web3-utils'
+import { config } from '@celo/protocol/migrationsConfig'
+import { toFixed } from '@celo/utils/lib/fixidity'
+import {
+  ExchangeEURInstance,
+  FreezerInstance,
+  ReserveInstance,
+  StableTokenEURInstance,
+} from 'types'
+
+const initializeArgs = async (): Promise<any[]> => {
+  const stableTokenEUR: StableTokenEURInstance = await getDeployedProxiedContract<
+    StableTokenEURInstance
+  >('StableTokenEUR', artifacts)
+  return [
+    // TODO replace this with specific
```

Comment:
What's this TODO for? Should we create a ticket? (channeling @nambrot 😄)

Reply:
cc @martinvol but I think we can simply remove